DKGRituals/src/utils/helper.ts

```diff
@@ -0,0 +1,60 @@
+import { Bytes } from "@graphprotocol/graph-ts";
```

Comment:
Feels like what is in here `helper.ts` can be moved to `utils.ts`.

Any reason for having separate files?

Reply:
I'm not strongly opiniated about this, just followed the same code structure pattern used in tBTC subgraphs, at first it made sense using helper.ts for the functions that directly manipulate the graph entities, acting as a helper extension to the mapping handlers. While in utils we can find more generic functions mainly used for transforming data types/values.